### PR TITLE
Feature/lesson6/homework

### PR DESCRIPTION
**Pull request by Feityanka**
Hello! Here is my homework. And since I don't have my own laptop, should I re-made the task we have done on our previous lesson and push here?
And yeah. I actually have made the last one stroke empty, but git doesn't actually want to save it and deletes...

### CODE REVIEW

File: Lesson6/decorator.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+def some_decorator(what_time):
+    def the_wrapper():
+        from datetime import datetime
```
Comment (nicilytop-tms) [inline Lesson6/decorator.py]: lets move this import to the begining of the file

File: Lesson6/isdigit.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+num = input()
```
Comment (nicilytop-tms) [inline Lesson6/isdigit.py]: so.. if I pass  `-.777` - I get **You have written a positive decimal number: -.777!**, but in isnt positive

**Review by nicilytop-tms — CHANGES_REQUESTED** [review]

File: Lesson6/isdigit.py (hunk 2 above)
Comment (Feityanka) [reply Lesson6/isdigit.py]: Oh, right. I will fix this!

File: Lesson6/isdigit.py [hunk 3]
```diff
@@ -1,10 +1,33 @@
-num = input()
+def check_digit_in_string(n):
+    flag = True
```
Comment (nicilytop-tms) [inline Lesson6/isdigit.py:2]: Oooops.. flag again? 😄 I don't even know.. Dmitry helped you or u helped Dmitry 🥲

File: Lesson6/isdigit.py [hunk 4]
```diff
@@ -1,10 +1,33 @@
...
-if num.isdigit():
-    print(f'You have written a positive number: {num}!')
-elif float(num) % 1 != 0:
-    print(f'You have written a positive decimal number: {num}!')
-elif (float(num) < 0) % 1 != 0:
-    print(f'You have written a negative decimal number: {num}!')
-else:
-    print(f'You have written a negative number: {num}!')
+
+check_digit_in_string(input())
```
Comment (nicilytop-tms) [inline Lesson6/isdigit.py:33]: dont have empty line at the end file.. PEP8

File: Lesson7/NewFile.txt
Comment (nicilytop-tms) [inline Lesson7/NewFile.txt:1]: we dont need this file

File: Lesson7/some.csv
Comment (nicilytop-tms) [inline Lesson7/some.csv:1]: and this file also..

**Review by nicilytop-tms — CHANGES_REQUESTED** [review]

**Pull request merged**